src/util/timemanager.cpp

```diff
@@ -412,6 +418,96 @@ void TimeManager::setDeltaTimeSteps(std::vector<double> deltaTimes) {
 
     _deltaTimeSteps = std::move(deltaTimes);
     _deltaTimeStepsChanged = true;
+
+    addDeltaTimesKeybindings();
+}
+
+void TimeManager::addDeltaTimesKeybindings() {
+    constexpr const std::array<Key, 10> Keys = {
+        Key::Num1,
+        Key::Num2,
+        Key::Num3,
+        Key::Num4,
+        Key::Num5,
+        Key::Num6,
+        Key::Num7,
+        Key::Num8,
+        Key::Num9,
+        Key::Num0
+    };
+
+    // First, clear any previous keybinds
+    for (const KeyWithModifier& kb : _deltaTimeStepKeybindings) {
+        global::keybindingManager.removeKeyBinding(kb);
```

Comment:
Could this remove user-keybindings?  If so, it might be better to check if there is a keybind and warn if there is, before removing it?